DROD/RoomWidget.cpp

```diff
@@ -1532,6 +1532,19 @@ void CRoomWidget::DisplayRoomCoordSubtitle(const UINT wX, const UINT wY)
 		}
 	}
 
+	// T-Covered tiles.
+	const UINT tCoveredTile = this->pRoom->GetCoveredTSquare(wX, wY);
+	if (tCoveredTile != T_EMPTY)
+	{
+		switch (tCoveredTile)
+		{
+		case T_TOKEN:
+			mid = GetTokenMID(this->pRoom->GetTParam(wX, wY)); break;
+			default: mid = TILE_MID[tCoveredTile]; break;
```

Comment:
Indentation here is tricky to read, but code is functionally correct.

Reply:
You're right. I've updated this to be consistend with the switch case formatting used throughout the function